atlasdb-commons/src/main/java/com/palantir/common/visitor/Visitor.java

```diff
@@ -13,14 +13,13 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package com.palantir.util;
-
+package com.palantir.common.visitor;
```

Comment:
this is API breaking, assuming @rjullman you're going to fix all other uses of this?